.github/workflows/python_lightning_kokkos_test_durations

```diff
@@ -0,0 +1,4523 @@
+{
```

Comment:
Should this file be `name.json`, or something else?

Reply:
Yes, I'll rename with a `json` extension.